stories/button/button.stories.js

```diff
@@ -274,34 +274,42 @@ export const buttonStates = () => `
 <button class="fd-button fd-button--emphasized">Normal State</button>
 <button class="fd-button fd-button--emphasized is-selected" aria-selected="true">Selected State</button>
 <button class="fd-button fd-button--emphasized" aria-disabled="true" disabled>Disabled State</button>
+<button aria-disabled="true" aria-describedby="fd-ONXuqucVcF" class="fd-button fd-button--emphasized is-disabled" type="button">Disabled Focusable</button>
```

Comment:
I would also add the instructions to this story so that the `aria-describedby` can be read out.

Reply:
Gotcha. I will add the hidden instructions so devs using a screen reader can see the intended use.